### PR TITLE
label all dokku containers

### PR DESCRIPTION
**Pull request by michaelshobbs**
refs: #1220

### CODE REVIEW

File: docs/development/plugin-creation.md [hunk 1]
```diff
@@ -90,3 +90,4 @@ A few notes:
   ```
 - From time to time you may want to allow other plugins access to (some of) your plugin's functionality. You can expose this by including a `functions` file in your plugin for others to source. Consider all functions in that file to be publicly accessible by other plugins. Any functions not wished to be made "public" should reside within your plugin trigger or commands files.
 - As of 0.4.0, we allow image tagging and deployment of said tagged images. Therefore, hard-coding of `$IMAGE` as `dokku/$APP` is no longer sufficient. Instead, for non `pre/post-build-*` plugins, use `get_running_image_tag()` & `get_app_image_name()` as sourced from common/functions. See the [plugin triggers](http://dokku.viewdocs.io/dokku/development/plugin-triggers) doc for examples.
+- As of 0.5.0, we will be using container labels to help cleanup intermediate containers with `dokku cleanup`. If calling `docker run`, include `$DOKKU_GLOBAL_RUN_ARGS`. This will ensure you intermediate containers labelled correctly.
```
Comment (josegonzalez) [inline docs/development/plugin-creation.md]: - `we will be using` => `we use`
- `If calling` => `If manually calling` (should we provide a wrapper for this?).

File: dokku [hunk 2]
```diff
@@ -29,6 +29,9 @@ export DOKKU_VALID_EXIT=0
 export DOKKU_LOGS_DIR=${DOKKU_LOGS_DIR:="/var/log/dokku"}
 export DOKKU_EVENTS_LOGFILE=${DOKKU_EVENTS_LOGFILE:="$DOKKU_LOGS_DIR/events.log"}
 
+export DOKKU_CONTAINER_LABEL=dokku
+export DOKKU_GLOBAL_RUN_ARGS="--label=$DOKKU_CONTAINER_LABEL"
```
Comment (josegonzalez) [inline dokku]: Might also be useful for partial swarm support.

Comment (michaelshobbs) [inline dokku]: :+1:

File: docs/development/plugin-creation.md (hunk 1 above)
Comment (michaelshobbs) [inline docs/development/plugin-creation.md]: I'm on the fence on the wrapper topic. I thought about it but meh

**Pull request merged**